### PR TITLE
fix: use xDAI and xBZZ on Gnosis chain

### PR DESCRIPTION
**Pull request by vojtechsimetka**

### CODE REVIEW

**Comment by bee-worker** [comment]
## 🐝 PR preview in Swarm

**Preview URL:** https://bah5acgzazyacxxpomhnngjireyqg5foftvs42jsb3frwfpgxkc5cgfqkhstq.bzz.link
**Swarm Hash:** `ce002bddee61dad3251126206e95c59d65cd2641d96362bcd750ba23160a3ca7`
**Commit Hash:** f45921e361ca915325091ffb62a25b51c0b9f049
**Commit Message:** fix: use xDAI on Gnosis chain

File: src/pages/status/SetupSteps/ChequebookDeployFund.tsx [hunk 1]
```diff
@@ -24,23 +24,23 @@ const ChequebookDeployFund = (): ReactElement | null => {
...
           Your chequebook is not funded. Please deposit some xBZZ to your chequebook address. You may need to aquire BZZ
-          (e.g. <a href="https://bzz.exchange/">bzz.exchange</a>) and bridge it to the xDai network through the{' '}
+          (e.g. <a href="https://bzz.exchange/">bzz.exchange</a>) and bridge it to the Gnosis chain network through the{' '}
           <a href="https://omni.xdaichain.com/bridge">omni bridge</a>. To pay the transaction fees, you will also need
-          xDAI token. You can purchase DAI on the network and bridge it to xDai network through the{' '}
-          <a href="https://bridge.xdaichain.com/">xDai Bridge</a>. See the{' '}
-          <a href="https://www.xdaichain.com/#xdai-stable-chain">official xDai website</a> for more information.
+          xDAI token. You can purchase DAI on the Ethereum mainnet network and bridge it to Gnosis chain network through
+          the <a href="https://bridge.xdaichain.com/">xDai Bridge</a>. See the{' '}
+          <a href="https://www.xdaichain.com/#xdai-stable-chain">official Gnosis chain website</a> for more information.
```
Comment (agazso) [inline src/pages/status/SetupSteps/ChequebookDeployFund.tsx:31]: The official website is https://www.gnosischain.com/evm

The link you provided gives me 500

File: src/pages/top-up/index.tsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ import { SwarmDivider } from '../../components/SwarmDivider'
 import { Context } from '../../providers/Bee'
 import { TopUpProgressIndicator } from './TopUpProgressIndicator'
 
-const MINIMUM_XDAI = '0.5'
+const MINIMUM_xDAI = '0.5'
```
Comment (agazso) [inline src/pages/top-up/index.tsx:14]: In constants I would not use lowercase

Comment (vojtechsimetka) [reply src/pages/top-up/index.tsx]: I overlooked this one, fixed.

**Review by agazso — APPROVED** [review]

**Pull request merged**